openshift/minishift-addons/rhche/templates/rh-che.app.yaml

```diff
@@ -62,6 +73,12 @@ objects:
         labels:
           app: rhche
       spec:
+        initContainers:
+          - image: ' '
```

Comment:
> Why is this field empty?

Because the image is defined later in the trigger definition: https://github.com/redhat-developer/rh-che/pull/711/files/6354c831ca5b3c9a4f131ce983eb6e9101e4bdbe#diff-f33365d04d532869c54d961801882a5eR294 

> Anyway why aren't you executing the keycloak configurator when the rh-che prerequisites are deployed (i.e. keycloak-template.yaml)?

Several reasons led me to do this:
- to ensure Keycloak is correctly configured *before* the RhChe server starts
- What would trigger the start of this command ? We might do this in a post hook in the Keycloak deployment config, but then we should add the corresponding code in the docker image of the Keycloak main container. This implies creating a dedicated Keycloak docker image for RhChe which would derive from the `eclipse/che-keycloak` image... Is it what we want ?